packages/kamino-sdk/src/Kamino.ts

```diff
@@ -2305,6 +2305,16 @@ export class Kamino {
     return updateStrategyConfig(args, accounts);
   };
 
+  getStrategyRebalanceParams = async (strategy: PublicKey | StrategyWithAddress): Promise<number[]> => {
+    const strategyState = await this.getStrategyStateIfNotFetched(strategy);
+    if (!strategyState) {
```

Comment:
I don’t think this check is needed, afaik the above method already does that and guarantees it will return strategy state or throw